quartz/components/renderPage.tsx

```diff
@@ -220,19 +220,19 @@ export function renderPage(
         <div id="quartz-root" class="page">
           <Body {...componentData}>
             {LeftComponent}
-            <div class="center">
-              <div class="page-header">
```

Comment:
Actually I think it still makes sense to put `page-header` into `center`

see https://quantum-engineering-polito.github.io/CourseNotes/FTP/03_Clean-Room-Technology/Clean-Room-Technology for example here (they are using page-header for their own component)